Submodels/MTSOL_call.py

```
@@ -607,19 +619,15 @@ def GenerateCrashOutputs(self,
         """
-            
-        # Define the filepath
-        file_path = f'forces.{self.analysis_name}'
 
         # To avoid mis-interpretation of files, delete the flowfield and boundary layer files, if they exist. 
         # For a crash output, these files are not needed, as we only use the forces.xxx file. 
-        if os.path.exists(f"flowfield.{self.analysis_name}"):
-            os.remove(f"flowfield.{self.analysis_name}")
-        if os.path.exists(f"boundary_layer.{self.analysis_name}"):
-            os.remove(f"boundary_layer.{self.analysis_name}")
+        for file in self.filepaths and file != self.filepaths['forces']:
```

Ruff (0.8.2)

Loop control variable `file` overrides iterable it iterates

(B020)